var/spack/repos/builtin/packages/foam-extend/package.py

```diff
@@ -111,189 +107,77 @@ class FoamExtend(Package):
     depends_on('scotch~metis+mpi', when='+ptscotch')
     depends_on('metis@5:',         when='+metis')
     depends_on('parmetis',         when='+parmetis')
-    depends_on('parmgridgen',      when='+parmgridgen')
+    # mgridgen is statically linked
+    depends_on('parmgridgen',      when='+parmgridgen', type='build')
```

Comment:
Is there any reason not to use the default dependency type? Just because OpenFOAM is special doesn't mean it doesn't link to that library.